### DISCUSSION TITLE
Is shopinvader suitable to work with odoo.sh?

### DISCUSSION THREAD

**Discussion by Mat-moran** [shopinvader/odoo-shopinvader#1327] [Q&A] [answered]
Is shopinvader suitable to work with odoo.sh?

😄

**Comment by david-fleischmann** [comment]
Would be interesting for me to know too :)

**Comment by simahawk** [comment, marked as answer]
In general I'd say why not? 
First issue that comes to my mind: q.job is heavily used and they don't run so efficiently on odoo.sh (see limitations of https://github.com/OCA/queue/tree/16.0/queue_job_cron_jobrunner).

Anyway, you could give it a try and see how it works :wink:

**Reply by hparfr** [reply]
I did a test to export products to elastic  with queue_job_cron_jobrunner on sh.
It's a bit slow but it worked.

I didn't tried to request the api.  One point of attention is if the HTTP headers are well preserved. Auth_api_key and website_unique_key (if multiple shopinvader backend) both needs headers.
